s3/s3.go

```diff
@@ -51,14 +52,19 @@ type s3client struct {
 }
 
 // GetAssumeRoleCredentials gets Assumed role credentials
-func GetAssumeRoleCredentials(opts S3ClientOpts) (*credentials.Credentials, error) {
+func GetAwsCredentials(opts S3ClientOpts) (*credentials.Credentials, error) {
```

Comment:
should you be changing the name of an exported func?

Reply:
I re-worked to to not change the exported function and it better follows the code thats there.